### PR TITLE
fix: insert link in description field

### LINKED ISSUE #5017: Wizard Step 1 / Public Event Page: Description Cannot add Links
For spam reasons I think we did not support links in the event description previously. However, it turns out that it is important for events to be able to add links into the description.

Therefore please provide the option to add links and linkify them in the public event page.

Screenshot from 2020-09-08 12-10-00

### PR DESCRIPTION
**Pull request by maze-runnar**
Fixes #5017

Screenshot from 2020-09-09 13-01-38
Screenshot from 2020-09-09 13-00-44
Screenshot from 2020-09-09 13-00-27

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Review by snitin315 — COMMENTED** [review]
Works perfect but there is one issue when we create a new event the `createLink` popup is active by default and not hidden.

Screenshot at 2020-09-09 17-18-02

**Comment by maze-runnar** [comment]
> Works perfect but there is one issue when we create a new event the `createLink` popup is active by default and not hidden.
> 
> Screenshot at 2020-09-09 17-18-02

Any suggestions for this, it is just by default.

File: app/templates/components/widgets/forms/rich-text-editor.hbs [hunk 1]
```diff
@@ -13,5 +13,15 @@
     <a href="#" class="ui button" data-content="{{t 'Redo (Ctrl+Y)'}}" data-wysihtml5-command="redo"><i class="repeat icon"></i></a>
     <a href="#" class="ui button" data-content="{{t 'Line Break'}}" data-wysihtml5-command="insertLineBreak"><i class="level down icon"></i></a>
   </div>
+  <div class="ui icon buttons">
+    <a href="#" class="ui button" data-wysihtml5-command="createLink" data-content="{{t 'Insert Link'}}" ><i class="linkify icon"></i></a>
+  </div>
+    <div data-wysihtml5-dialog="createLink" class="mb-8">
```
Comment (snitin315) [inline app/templates/components/widgets/forms/rich-text-editor.hbs:19]: Can you check if this works?

```suggestion
    <div data-wysihtml5-dialog="createLink" class="mb-8" style="display: 'none' ">
```

Refers https://github.com/Voog/wysihtml/blob/3f7a34a2721813c81c3b2205a9bfff8b8e472eff/examples/advanced_div.html#L262-L268

Comment (maze-runnar) [reply app/templates/components/widgets/forms/rich-text-editor.hbs]: thanks man :+1:

**Review by snitin315 — APPROVED** [review]
Good job

**Review by iamareebjamal — APPROVED** [review]

**Pull request merged**